gnomad/assessment/summary_stats.py

```diff
@@ -355,6 +356,185 @@ def get_tx_expression_expr(
     )
 
 
+def get_summary_stats_variant_filter_expr(
+    t: Union[hl.Table, hl.MatrixTable],
+    filter_lcr: bool = False,
+    filter_expr: hl.expr.SetExpression = None,
+    freq_expr: hl.expr.SetExpression = None,
+    max_af: Optional[Union[float, List[float]]] = None,
+    min_an_proportion: Optional[float] = None,
+    collapse_filters: bool = False,
+) -> Union[hl.expr.BooleanExpression, Dict[str, hl.expr.BooleanExpression]]:
+    """
+    Generate variant filtering expression for summary stats.
+
+    :param t: Input Table/MatrixTable.
+    :param filter_lcr: Whether to filter out low confidence regions. Default is False.
+    :param filter_expr: SetExpression containing variant filters. Default is None.
+    :param freq_expr: SetExpression containing frequency information. Default is None.
+    :param max_af: Maximum allele frequency cutoff(s). Can be a single float or a list
+        of floats. Default is None.
+    :param min_an_proportion: Minimum allele number proportion (used as a proxy for
+        call rate). Default is None.
+    :param collapse_filters: Whether to collapse all filters into a single expression.
+        Default is False.
+    :return: BooleanExpression or Dict of BooleanExpressions for filtering variants.
+    """
+    if max_af is not None and freq_expr is None:
+        raise ValueError("Frequency expression must be provided when filtering by AF!")
+
+    log_list = []
+    ss_filter_expr = {}
+    if filter_lcr:
+        log_list.append("variants in low confidence regions")
+        ss_filter_expr["no_lcr"] = low_conf_regions_expr(t.locus, filter_decoy=False)
+    if filter_expr is not None:
+        log_list.append("variants that pass all variant QC filters")
+        ss_filter_expr["variant_qc_pass"] = hl.len(filter_expr) == 0
+    if max_af is not None:
+        if isinstance(max_af, float):
+            max_af = [max_af]
+        for af in max_af:
+            log_list.append(f"variants with (AF < {af:.2e})")
+            ss_filter_expr[f"max_af_{af}"] = freq_expr < af
+    if min_an_proportion is not None:
+        log_list.append(
+            "variants that meet a minimum call rate of %.2f (using AN as a call rate "
+            "proxy)" % min_an_proportion,
+        )
+        ss_filter_expr[f"min_an_{max_af}"] = get_an_criteria(
```

Comment:
```suggestion
        ss_filter_expr[f"min_an_{min_an_proportion}"] = get_an_criteria(
```

unless I'm unclear about what this is doing here?